frontend/packages/metal3-plugin/src/plugin.ts

```diff
@@ -158,6 +161,24 @@ const plugin: Plugin<ConsumedExtensions> = [
       required: [FLAGS.BAREMETAL, METAL3_FLAG],
     },
   },
+  {
+    type: 'Dashboards/Overview/Activity/Resource',
+    properties: {
+      k8sResource: {
+        kind: referenceForModel(BareMetalHostModel),
+        prop: 'bmhs',
```

Comment:
```suggestion
        prop: 'bmhs',
        isList: true,
```

`isList: true` is always required otherwise the code in activity-card.tsx breaks as it always expects `resource.data` to be an array. It would be nice to catch this in the `Dashboards/Overview/Activity/Resource` type definition.

Another option would be to update `OngoingActivity` component code to accommodate for cases when a `resource.data` is an object rather than array.

(see activity-card.tsx comment)

Reply:
type definition for `k8sResource` now enforces `isList: true`